numpy/core/records.py

```diff
@@ -595,8 +595,30 @@ def _deprecate_shape_0_as_None(shape):
 
 def fromarrays(arrayList, dtype=None, shape=None, formats=None,
                names=None, titles=None, aligned=False, byteorder=None):
-    """ create a record array from a (flat) list of arrays
+    """Create a record array from a (flat) list of arrays
 
+    Parameters
+    ----------
+    arrayList : sequence
+        Iterable container of array-like objects (such as lists, tuples,
+        and ndarrays).
```

Comment:
```suggestion
    arrayList : list or tuple
        List of array-like objects (such as lists, tuples,
        and ndarrays).
```
The original wording is technically correct, but I would advocate for stronger wording here to discourage the use of multi-D `ndarray` for the `arrayList` parameter. See #15849 for related discussion.